beaglebone/beaglebone-udc.txt

```diff
@@ -46,16 +46,19 @@ LA-RaCtrl:CO-PSCtrl-TB2     LA-RaPS06:PS-UDC-TB4    LA-RaPS06:PS-UDC-TB5    LA-R
 # ______ TS ______
 
 # --- TS Dipole ---
-LA-RaPS02:CO-PSCtrl-TS      LA-RaPS02:PS-UDC-TS1    LA-RaPS02:PS-UDC-TS2
+LA-RaPS02:CO-PSCtrl-TS      LA-RaPS02:PS-UDC-TS
```

Comment:
@xresende, I think there's a `1` missing at the end of the `UDC` definition:

`LA-RaPS02:PS-UDC-TS1`

Reply:
done. thanks!